### DISCUSSION TITLE
Add Fields Query Params Receipt Resource

### DISCUSSION THREAD

**Discussion by Esatyilmaz0** [etsy/open-api#830] [Ideas]
Hello, today when I tried to pull all the receipts belonging to my store (36000 units), each request took 10 seconds on average and it took me 1 hour to pull all of them. Therefore, the idea that came to my mind is that if we add a fields query parameter to the receipts resource and only get the fields we need, the results in such operations will experience a significant performance increase. What are you thinking?

**Comment by Gareth064** [comment]
#209 discuss this feature. But nothing has been updated on it in 1.5 years

**Comment by mk148a** [comment]
> Hello, today when I tried to pull all the receipts belonging to my store (36000 units), each request took 10 seconds on average and it took me 1 hour to pull all of them. Therefore, the idea that came to my mind is that if we add a fields query parameter to the receipts resource and only get the fields we need, the results in such operations will experience a significant performance increase. What are you thinking?

You can use paralel requests for accerate
